Core/Core/Files/FileList/FileListViewController.swift

```diff
@@ -211,7 +211,8 @@ public class FileListViewController: UIViewController, ColoredNavViewProtocol {
     }
 
     func showDeleteAlert(name: String, handler: @escaping ((UIAlertAction) -> Void)) {
-        let alert = UIAlertController(title: NSLocalizedString("Are you sure you want to delete \(name)?", bundle: .core, comment: ""), message: nil, preferredStyle: .alert)
+        let title = String.localizedStringWithFormat(NSLocalizedString("Are you sure you want to delete %@?", bundle: .core, comment: ""), name)
```

Comment:
This looks good, but in the future, we have an overload of NSLocalizedString that will assume bundle is the current target (.core for here) and an empty comment. So this could omit `, bundle: .core, comment: ""`.